setup.py

```diff
@@ -5,7 +5,9 @@
     "pyyaml~=6.0",
     "wasabi",
     "numpy",
-    "cloudpickle>=1.6"
+    "cloudpickle>=1.6",
+    "gym~=0.21",
```

Comment:
this prevents hf hub sb3 from using it with gymnasium/sb3, no? is it needed only for the tests?

Reply:
I think right now `huggingface_sb3` does not support `gymnasium` anyways (see #30).

Reply:
Hey there 👋 I'm finishing the gymnasium integration can we remove this requirement? (gym one)

The PR: https://github.com/huggingface/huggingface_sb3/compare/gymnasium-v2